pep-0634.rst

```diff
@@ -0,0 +1,696 @@
+PEP: 634
+Title: Structural Pattern Matching: Specification
+Version: $Revision$
+Last-Modified: $Date$
+Author: Brandt Bucher <brandtbucher@gmail.com>,
+        Guido van Rossum <guido@python.org>
+BDFL-Delegate:
+Discussions-To: Python-Dev <python-dev@python.org>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 12-Sep-2020
+Python-Version: 3.10
+Post-History:
+Replaces: 622
+Resolution:
+
+
+Abstract
+========
+
+TODO: Do we write ``match`` statement or just match statement?
+Currently we're inconsistent.
```

Comment:
I'd like it if we used "`match`" to refer to the actual syntax construct and "match" otherwise. For example: "If the `match` statement fails to match any of the patterns...".

Reply:
Okay, that means whenever it says "match statement" it should become "`match` statement". Okay.